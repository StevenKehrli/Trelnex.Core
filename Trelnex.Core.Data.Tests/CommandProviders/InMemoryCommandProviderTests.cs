namespace Trelnex.Core.Data.Tests.CommandProviders;

public class InMemoryCommandProviderTests : CommandProviderTests
{
    [SetUp]
    public async Task TestFixtureSetup()
    {
        // create our command provider
        var factory = await InMemoryCommandProviderFactory.Create();

        _commandProvider =
            factory.Create<ITestItem, TestItem>(
                typeName: "test-item",
                TestItem.Validator,
                CommandOperations.All);
    }
}
