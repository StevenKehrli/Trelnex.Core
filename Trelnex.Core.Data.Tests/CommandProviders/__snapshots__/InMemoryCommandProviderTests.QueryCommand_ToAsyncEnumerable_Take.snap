[
  {
    "Item": {
      "Id": "a1a59dff-0c00-498c-8190-9ee28395f7ed",
      "PartitionKey": "ed834fb0-9294-4b8f-ad0a-77a84e5feea9",
      "TypeName": "test-item",
      "CreatedDate": "2025-01-07T00:54:42.097493Z",
      "UpdatedDate": "2025-01-07T00:54:42.097493Z",
      "DeletedDate": null,
      "IsDeleted": null,
      "ETag": "54479c79-f1cb-40ee-8133-919d9b877b8f",
      "Message": "Message #1"
    }
  }
]
